modules/tinymce/src/plugins/table/main/ts/actions/Clipboard.ts

```diff
@@ -50,12 +50,13 @@ const registerEvents = (editor: Editor, selections: Selections, actions: TableAc
   });
 
   editor.on('BeforeSetContent', (e) => {
-    if (e.selection === true && e.paste === true) {
+    const content = e.content;
+    if (e.selection === true && e.paste === true && Type.isString(content)) {
       const selectedCells = TableSelection.getCellsFromSelection(selections);
       Arr.head(selectedCells).each((cell) => {
         TableLookup.table(cell).each((table) => {
 
-          const elements = Arr.filter(SugarElements.fromHtml(e.content), (content) => {
+          const elements = Arr.filter(SugarElements.fromHtml(content), (content) => {
```

Comment:
We should try to avoid shadowing this `content` variable name — either change the name in the outer scope (the variable you added in this PR) or in the inner scope (not code you've touched in this PR directly, but might be easier to find a new name here)